prettier.config.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
```

Comment:
Not overly bothered by this, but has this snuck in? 😮

Reply:
yes, for some reason my prettier started to use different settings, so I realized the repo doesn't have explicit prettier rules, so I added them :)